optaplanner-integration/optaplanner-spring-boot-autoconfigure/src/main/java/org/optaplanner/spring/boot/autoconfigure/TerminationProperties.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.spring.boot.autoconfigure;
+
+import java.time.Duration;
+
+public class TerminationProperties {
+
+    /**
+     * How long the solver can run. Uses ISO 8601 format, see {@link Duration}.
+     * For example: "30S" is 30 seconds. "5M" is 5 minutes. "2H" is 2 hours.
```

Comment:
The conversion from string to a `Duration` is handled by [Spring Boot](https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration). ISO 8601 is one of the supported formats. I recommend using the more readable format (30s, 5m, 2h30m etc.).

Reply:
Yes, absolutely true for UX.